packages/@webex/plugin-meetings/src/statsAnalyzer/index.ts

```diff
@@ -103,7 +105,20 @@ export class StatsAnalyzer extends EventsScope {
     this.correlationId = config.correlationId;
     this.mqaSentCount = -1;
     this.lastMqaDataSent = {};
-    this.lastEmittedStartStopEvent = {};
+    this.lastEmittedStartStopEvent = {
+      audio: {
```

Comment:
if we do this initialization here now, then we don't need the code in lines 466-468

Reply:
Removed line 466-468